src/TextLink/index.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import * as React from "react";
+import { darken } from "polished";
+import * as tokens from "orbit-design-token";
+
+type Props = {
+  title: string,
+  url: string,
+  onClick: (e: SyntheticEvent<HTMLButtonElement>) => void,
```

Comment:
accepting `e` param should be optional

Reply:
`(SyntheticEvent<HTMLButtonElement>) => any`

Reply:
OK